pkgs/misc/cups/drivers/foomatic-db-ppds/default.nix

```
@@ -0,0 +1,128 @@
+{ lib
+, foomatic-db
+, foomatic-db-nonfree
+, buildEnv
+, writers
+, foomatic-db-engine
+, perlPackages
+, stdenv
+, cups-filters
+, ghostscript
+, netpbm
+, perl
+, psutils
+, patch-ppd-files
+}:
+
+let
+  foomatic-db-packages = [ foomatic-db foomatic-db-nonfree ];
+
+  foomatic-db-combined = buildEnv {
+    name = "foomatic-db-combined";
+    paths = foomatic-db-packages;
+    pathsToLink = [ "/share/foomatic" ];
+    postBuild = ''
```

If we'll leave things as they are regarding this new package, I think it'd be nice to add a comment like this (assuming I understood what's going on correctly):

```suggestion
    # `foomatic-db-combined` is a nativeBuildInput of `foomatic-db-ppds`. Thanks to this
    # setup hook, our `foomatic-ppdfiles.pl` script knows where to find the database to
    # generate ppd files from.
```

---

Yes, I appended a corresponding comment.